README.md

```diff
@@ -1,2 +1,44 @@
 # My_Portfolio
-A little about me - **A website built in HTML and CSS**
+**A little about me - A website built in HTML and CSS**
+
+
+## Built With
+
+- HTML & CSS
+- No Framework
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Deployment
+
+Deployed on Github
+
+## Authors
+
+👤 Yordanos Temesgen
+
+- GitHub: [@yordinia](https://github.com/yordinia)
+- Twitter: [@yordinia](https://twitter.com/yordinia)
+- LinkedIn: [Yordanos-Temesgen](https://linkedin.com/in/yordanos-temesgen-251b6a202
+)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] The MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯